components/tests/ui/testcases/web/post_test.txt

```diff
@@ -114,3 +235,246 @@ Test Chgrp
+    ${datasetNode}=                             Wait For Dataset Node                                             ${datasetId}    
+    ${datasetNode1}=                            Wait For Dataset Node                                             ${did}
+
+    Wait Until Element is Visible               xpath=//li[@id='${datasetNode}']//li[@data-id='${imageId}']/a                                
+    Drag And Drop                               xpath=//li[@id='${datasetNode}']//li[@data-id='${imageId}']/a     xpath=//li[@id='${datasetNode1}']/a
+    Select Dataset By Id                        ${did}
+    Wait Until Element is Visible               xpath=//li[@id='${datasetNode1}']//li[@data-id='${imageId}'] 
+
+    #Bring database state back to original
+    Select Image By Id                          ${imageId}
+    Wait Until Element is Visible               xpath=//li[@id='${datasetNode1}']//li[@data-id='${imageId}']/a
+    Drag And Drop                               xpath=//li[@id='${datasetNode1}']//li[@data-id='${imageId}']/a    xpath=//li[@id='${datasetNode}']/a
+    Select Dataset By Id                        ${datasetId}
+    Wait Until Element is Visible               xpath=//li[@id='${datasetNode}']//li[@data-id='${imageId}']                          
+
+Test Drag and Drop Orphaned Image
+    
+    Wait Until Keyword Succeeds                 ${TIMEOUT}    ${INTERVAL}     Reload Page
+    Set Selenium Speed                          .3
+    Select Experimenter
+    ${did}=                                     Select First Dataset
+    ${datasetNode}=                             Wait For Dataset Node                                           ${did}
+
+    ${nodeId}=                                  Select Orphaned Images Section
+    ${imageId}=                                 Select First Orphaned Image
+    ${imageNode}=                               Wait For Image Node                                             ${imageId}
+
+    Wait Until Element is Visible               xpath=//li[@id='${nodeId}']//li[@id='${imageNode}']/a
+    Drag And Drop                               xpath=//li[@id='${nodeId}']//li[@id='${imageNode}']/a           xpath=//li[@id='${datasetNode}']/a
+    Select Dataset By Id                        ${did}
+    Dataset Should Contain Image                ${imageId}     ${did} 
+
+Test Drag and Drop Orphaned Dataset
+       
+    Wait Until Keyword Succeeds                 ${TIMEOUT}    ${INTERVAL}     Reload Page
+    Set Selenium Speed                          .3 
+    Select Experimenter                            
+    ${pid}=                                     Select First Project With Children
+    Select Experimenter                         
+    ${datasetId}                                Create Dataset
+    Wait Until Keyword Succeeds                 ${TIMEOUT}    ${INTERVAL}     Click Element                     refreshButton 
```

Comment:
Waiting for the refresh action to complete, somehow this allows the click element to complete its action as well..